extension/scripts/features/cleanflight/ttCleanFlight.css

```diff
@@ -0,0 +1,5 @@
+.travel-agency-travelling .stage,
+.travel-agency-travelling .stage + hr,
+.travel-agency-travelling .popup-info {
+	display: var(--torntools-clean-flight-display) !important;
```

Comment:
I'm not a fan of this approach, as it gives a poor compatibility with other things that affect the flight page.

Reply:
It's great that you thought of compatibility with the travel table though.